app/simulation_test.go

```diff
@@ -118,6 +118,7 @@ func TestAppStateDeterminism(t *testing.T) {
 	config.OnOperation = true
 	config.AllInvariants = true
 
+	rand.Seed(time.Now().Unix())
```

Comment:
Is it for the simulation seed? Can't we have a flag for it?

Reply:
we use `rand.Int63()` below

We could use the seed flag, but why? We are not seeking repeatability, but rather find non-determinism. @aljo242 opinions?

I could use the `rand.Seed(simapp.FlagSeedValue)` in any case, if wanted